Packs/GroupIB_ThreatIntelligence&Attribution/pack_metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Group-IB Threat Intelligence & Attribution",
+    "description": "Fast receive incidents related to you, attribute them to adversaries to do instant response. Enrich your security with an enormous IOCs collection.",
```

Comment:
->"Group-IB Threat Intelligence & Attribution is a system for analyzing and attributing cyberattacks, threat hunting, and protecting network infrastructure based on data relating to adversary tactics, tools and activity. Use this pack to fast receive incidents related to you, attribute them to adversaries to do instant response, enrich your security with an enormous IOCs collection, and provide possibilities for manual investigation through Group-IB data via Cortex XSOAR interface"

Reply:
Done